### ISSUE TITLE
Getting INTERNAL_SERVER_ERROR on mlflow ui each time I log my model through pytorch.

### ISSUE THREAD

**Issue by shivam-soni** [mlflow/mlflow#2813]
I am using Google Colab and mounting my drive on my local system. Then in the same folder where my code is, I am running the mlflow ui. The example code works fine but any other program is not working. Any suggestions would help.

**Comment by sueann** [comment]
Could you provide a little more detail on your issue?

* Which example code works for you?
* Is what's not working `pytorch.log_model`? Are there other specific MLflow calls not working?
* What backend store are you using for tracking? Is it the default (filestore)?
* Did you specify a `--default-artifact-root`? If not, (the default is `./mlruns`, which I assume would be inside your mounted drive. It's possible that MLflow doesn't have the correct permissions.
* Do you see any error messages in your console from where you are running `mlflow ui`?

**Comment by shivam-soni** [comment]
Thank you for the response Sue.

1.  After running some more experiments what I have found is,  not only the example code but now any kind of code's runs are getting logged. But the main issue is, if I am logging more than three runs under same the experiment (in nested loops) , then I am getting the "INTERNAL_SERVER_ERROR ".

![Screenshot from 2020-05-16 13-08-54](https://user-images.githubusercontent.com/14237704/82113842-992cb300-9776-11ea-99f1-1f4b0f7429cf.png)

Actually, I am trying to tune hyper-parameters for my model, as shown in the screenshot above , there can be three possible combination of learning_rate, momentum and hidden_nodes. For these three possible runs I am able to log but more than that I get error.

2. No, there isn't any other problem with any specific MLflow calls except the one explained in point number 1.

3. & 4.) I have used http://localhost:5000 as well as http://127.0.0.1:5000, both of them shows the same problem. I didn't specify any "--default-artifact-root" and yes ./mlruns is inside the my mounted drive where I run mlflow ui through command line. 

5. I am getting some Deprecation Warning.
![Screenshot from 2020-05-16 13-43-01](https://user-images.githubusercontent.com/14237704/82114611-5d481c80-977b-11ea-833c-47a26f87c8c8.png)

Please let me know where am I making mistake.

**Comment by shivam-soni** [comment]
Also getting "[CRITICAL] WORKER TIMEOUT" error. I didn't even find any solution for this issue anywhere.
![Screenshot from 2020-05-19 13-52-11](https://user-images.githubusercontent.com/14237704/82303097-1d31a580-99d8-11ea-8f48-551eb6397565.png)

**Comment by ne-bo** [comment]
I have the same problem with CRITICAL WORKER TIMEOUT

**Comment by boc0** [comment]
_priority/backlog_? **This causes the UI to not work at all**. Please label this as a bug.

**Comment by dpaetzel** [comment]
I managed to fix my own critical worker timeout problems that occurred upon opening the UI using [this suggestion](
https://github.com/mlflow/mlflow/issues/925#issuecomment-648320135). The problem was iirc that when too many things are loaded by the UI workers, some default timeout is hit which is really low. Increasing the timeout made everything work again for me.

However, I don't use pytorch, maybe the problem is different.

**Comment by dbczumar** [comment]
Closing in favor of https://github.com/mlflow/mlflow/issues/925, which we will continue to track and work to address
